include/coupled_interface/coupled_interface.hpp

```diff
@@ -108,8 +108,9 @@ class coupled_interface {
   int npoints; ///< Number of quadrature points in the interface.
   specfem::assembly::interface_container<SelfMedium, CoupledMedium>
       interface_data; ///< Struct containing the coupling information.
-  specfem::assembly::simulation_field<WavefieldType> field; ///< Wavefield
-                                                            ///< object.
+  specfem::assembly::simulation_field<dimension, WavefieldType>
```

Comment:
```suggestion
specfem::assembly::simulation_field<dimension_tag, WavefieldType>
```

Needs to be updated above though

Reply:
Coupled interfaces would be at a later date right?